.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "regression/azure-rest-api-specs"]
```

Comment:
Could you pls update `readme.md` about how to run regression test locally?

Reply:
Thanks added